augly/image/transforms.py

```diff
@@ -211,7 +211,34 @@ def apply_transform(
         @returns: Augmented PIL Image
         """
         return F.brightness(image, factor=self.factor, metadata=metadata)
-
+    
+class Skew(BaseTransform):
+    def __init__(self, level: float = 0.5, axis: str = "x-axis", p: float = 1.0):
+        
+        """
+        @param level: Level of the operation that will be in [0, The maximum value of level]
+            
+        @param axis: Axis along which the image will be skewed; can be 'x-axis' or 'y-axis'
+        """
+        super().__init__(p)
+        self.level = level
+        self.axis = axis
+        
+    def apply_transform(
+        self, image: Image.Image, metadata: Optional[List[Dict[str, Any]]] = None
+    ) -> Image.Image:
+        """
+        Skewing of Image
```

Comment:
Comment should be the same as in functional.py
```suggestion
        Skew image with respect to X-axis or Y-axis
```